docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single node.md

```diff
@@ -0,0 +1,761 @@
+The tictactoe directory now contains two new files, tictactoe.wasm and tictactoe.abi.
+
+### Deploy the Smart Contract to the Testnet
+[Deploy the smart contract](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-deploy-a-smart-contract) on the single node testnet with the following commandline.
+
+In the smae directory as the generated wasm and ABI files run
+```shell
+cleos set contract host ./ tictactoe.wasm tictactoe.abi -p host@active
+```
+
+## Play The Game
+Now that the smart contract has been successfully deployed, [push smart contract actions](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-action) to the blockchain to play the game.  
+
+
+### Push 'create' to the single node testnet to create a game
+A game requires a host and a challenger. Use the accounts created earlier in the “Create the necessary accounts and key pairs” sectio of the tutorial for these. These accounts use arbitrary names. In this example assume the host has the account name of ‘mcazyfujecke’ and the challenger has the account name of ‘vswlkiegwdsk’.
+
+The create action takes two parameters, the "challenger" and the "host". The required payload in json format is:
+
+```json
+{
+  "challenger": "challenger",
+  "host": "host"
+}
+```
+
+Sign the push action with host@active’, the host of the game.
+
+```shell
+cleos push action tictactoe create '{"challenger":"challenger", "host":"host"}' --permission host@active
+```
+
+### Push 'move' to the single node testnet to make game moves
+Players make moves in turn by pushing ‘move’ actions to the blockchain. The host moves first, and each move must be signed by the appropriate account.
+
+The move action takes five parameters, the "challenger", the "host", the player who makes the move or "by" and "row" and "column" parameters to show where the marker is placed.
+
+The host makes the first move. The required payload in json format is:
+
+```json
+{
+  "challenger": "challenger",
+  "host": "host",
+  "by": "host",
+  "row": 0,
+  "column": 1
+}
+```
+
+Sign the push action with host@active’ - the host of the game.
+
+```shell
+cleos push action tictactoe move '{"challenger":"challenger", "host":"host", "by":"host", "row":0, "column":1}' --permission host@active
+```
+
+The challenger makes the second move. The required payload in json format is:
+
+```json
+{
+  "challenger": "challenger",
+  "host": "host",
+  "by": "challenger",
+  "row": 1,
+  "column": 1
+}
+```
+
+Sign the push action with challenger@active’ - the challenger.
+
+```shell
+cleos push action tictactoe move '{"challenger":"challenger", "host":"host", "by":"challenger", "row":1, "column":1}' --permission challenger@active
+```
+
+Continue to make moves until the game ends with a win or a draw.
+
+### Check game status 
+Look at the data in the multi index table to check the game status. 
+
+```shell
+cleos get table tictactoe host games
+```
+
+### Push "restart" to the Testnet to restart the Game
```

Comment:
To avoid confusion on whether using the EOSIO Testnet or a single node blockchain, it's probably better to use "smart contract" instead. That way, the content remains neutral, and more accurate actually.

Reply:
makes sense to use smart contract